commands/git/git.md

```diff
@@ -2,11 +2,14 @@
 Daily used git commands
 
 - [Clone a repository](git-clone-repo.md)
+- [Commit changes](git-commit.md)
 - [Create a new branch](git-new-branch.md)
 - [Pull changes](git-pull.md)
+- [Push changes](git-push.md)
 - [RM Cached](git-rm-cached.md)
-- [Stage and commit](git-stage-commit.md)
+- [Stage](git-stage.md)
 - [Stash](git-stash.md)
 - [Stash File](git-stash-file.md)
 - [Stash List](git-stash-list.md)
 - [Stash Revert](git-stash-revert.md)
+- [Check status](git-status.md)
```

Comment:
Can you move it up, to make the list in alphabetical order

Reply:
I kept the sorting as per the file names. Should I change it as per description?

Reply:
`md` files only shows the description not the file names, so it will be better to sort alphabetical order by description.
Does that makes sense?

Reply:
Sure. Make sense